commit 96fed1d2 breaks AMF encoding on Linux

## Description
I'm using ALVR with AMF encoding on Linux, but with latest master it always says '[ERROR]	error in encoder thread: Failed to find device path'.
Reverting https://github.com/alvr-org/ALVR/commit/96fed1d211364660979829a281b16a73eee2ef4e fixes the issue.

SteamVR command:
```
env ALVR_AMF_ICD=/usr/share/vulkan/icd.d/amd_pro_icd64.json %command%
```
ALVR Log:
```
08:45:37.575311613 | [INFO] | CEncoder client connected, pid 215684, cmdline /home/andy/.local/share/Steam/steamapps/common/SteamVR/bin/linux64/vrcompositor
08:45:37.596784622 | [INFO] | Using Vulkan device AMD Radeon RX 6900 XT
08:45:37.600178552 | [ERROR] | error in encoder thread: Failed to find device path

--- a/alvr/server/cpp/platform/linux/ffmpeg_helper.cpp
+++ b/alvr/server/cpp/platform/linux/ffmpeg_helper.cpp
@@ -198,9 +198,9 @@ alvr::VkContext::VkContext(const char *deviceName, const std::vector<const char*
     }
   }
   if (devicePath.empty()) {
-    throw std::runtime_error("Failed to find device path");
+    devicePath = "/dev/dri/renderD128";
   }
-  Info("Found device path %s", devicePath.c_str());
+  Info("Using device path %s", devicePath.c_str());
 
   if (drmContext) {
     ctx = av_hwdevice_ctx_alloc(AV_HWDEVICE_TYPE_DRM);

